sdk/cognitiveservices/azure-cognitiveservices-anomalydetector/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release History
 
+## 0.3.1 (Unreleased)
```

Comment:
Not 0.4.0?

Reply:
Laurent just updated the guidance here: https://github.com/Azure/azure-sdk-for-python/blob/main/doc/deprecation_process.md#124-2023-03-31